src/GraphQL.MicrosoftDI/GraphQLBuilder.cs

```diff
@@ -29,10 +29,11 @@ public class GraphQLBuilder : GraphQLBuilderBase, IServiceCollection
         /// <remarks>
         /// Registers various default services via <see cref="GraphQLBuilderBase.Initialize"/>.
```

Comment:
Add some notes here that configure will be called first and default services registration last.

Reply:
Done